src/main/java/org/openlmis/integration/dhis2/web/server/ServerController.java

```diff
@@ -57,21 +59,54 @@
 public class ServerController extends BaseController {
```

Comment:
As we put all the logic in the controller, not the service, maybe consider writing some unit tests to make sure the logic in the methods works is correct.

Reply:
I have tested these controllers in integration tests instead